.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+## 📑 Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## 🍱 Types of changes
+
+What types of changes does your code introduce to AEP?
+_Put an `x` in the boxes that apply_
+
+- [ ] Enhancement
```

Comment:
Should we also have a checkbox for adopting an AIP from google.aip.dev?  It might be interesting to distinguish that from an entirely new proposal.

Reply:
I haven't thought of this one. I fully agree it makes sense during the initial phase.